logbook-json/src/main/java/org/zalando/logbook/json/StringBuilderWriter.java

```diff
@@ -0,0 +1,125 @@
+package org.zalando.logbook.json;
+
+import java.io.Serializable;
+import java.io.Writer;
+
+/**
+ * {@link Writer} implementation that outputs to a {@link StringBuilder}.
+ * Adapted from Apache commons-io equivalent. 
+ * <p>
+ * <strong>NOTE:</strong> This implementation, as an alternative to
+ * <code>java.io.StringWriter</code>, provides an <i>un-synchronized</i>
+ * (i.e. for use in a single thread) implementation for better performance.
+ * For safe usage with multiple {@link Thread}s then
+ * <code>java.io.StringWriter</code> should be used.
+ *
+ */
+class StringBuilderWriter extends Writer implements Serializable {
```

Comment:
Why `Serializable`?

Reply:
It was in the original source code. I'll remove it.